packages/jaeger-ui/src/components/SearchTracePage/SearchForm.track.tsx

```diff
@@ -34,7 +34,7 @@ export const CATEGORY_SERVICE = `${FORM_CATEGORY_BASE}/serviceName`;
 export function trackFormInput(
   resultsLimit: number,
   operation: string,
-  tags: any,
+  tags: string[],
```

Comment:
could you point to where this is shown as array being passed?

Reply:
https://github.com/jaegertracing/jaeger-ui/blob/ec3aba07a6439800b7477e761a3feabd4e74d47c/packages/jaeger-ui/src/components/SearchTracePage/SearchForm.jsx#L246

This was the code that I was able to trace it to. Since all the parameters implicitly have the type `any` here, I couldn't derive any concrete inferences. But since the same is being used in the context of a form, I presume that it must be an array of `string`.

Reply:
I think it's just a string that gets parsed later. https://github.com/jaegertracing/jaeger-ui/blob/ec3aba07a6439800b7477e761a3feabd4e74d47c/packages/jaeger-ui/src/components/SearchTracePage/SearchForm.jsx#L255

Reply:
Got it!